atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractPersistentStreamStore.java

```diff
@@ -144,10 +144,14 @@ public Sha256Hash apply(StreamMetadata metadata) {
         return hashes;
     }
 
-    protected final StreamMetadata storeBlocksAndGetFinalMetadata(@Nullable Transaction t, long id, InputStream stream) {
+    protected final StreamMetadata storeBlocksAndGetFinalMetadata(@Nullable Transaction t, long id, InputStream stream, boolean computeHash) {
```

Comment:
It's also possible that we can just leave this out - it's just a performance optimization to avoid taking a hash that we'll just throw away. I haven't measured how much we actually save here.